app/scripts/services/SongPlayer.js

```diff
@@ -100,8 +110,24 @@
             currentSongIndex--;
             
             if (currentSongIndex < 0) {
-                currentBuzzObject.stop();
-                SongPlayer.currentSong.playing = null;
+                stopSong(song);
+            } else {
+                var song = currentAlbum.songs[currentSongIndex];
+                setSong(song);
+                playSong(song);
+            }
+        };
+                
+        /*
+        * @method SongPlayer.next
+        * @desc Plays the next song by incrementing the currentSongIndex
+        */
+        SongPlayer.next = function() {
+            var currentSongIndex = getSongIndex(SongPlayer.currentSong);
+            currentSongIndex++;
+            
+            if (currentSongIndex > currentAlbum.songs.length) {
```

Comment:
After I clicked next on the player bar button until there were no more songs, the song stopped playing but the pause button on the number 5 disappeared unlike when you hit previous until you run out of songs.

I updated line 129 by adding `- 1` to the end and the pause button now shows when I click next until I run out of songs.